### PR TITLE
LPS-126597 Follow up

### PR DESCRIPTION
**Pull request by martamedio**
Follow up: https://github.com/brianchandotcom/liferay-portal/pull/99989#issuecomment-801040627

cc/ @csierra

### CODE REVIEW

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/Error.java [hunk 1]
```diff
@@ -0,0 +1,340 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/Error.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/Shipment.java [hunk 2]
```diff
@@ -0,0 +1,880 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/Shipment.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/ShipmentItem.java [hunk 3]
```diff
@@ -0,0 +1,544 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/ShipmentItem.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/ShippingAddress.java [hunk 4]
```diff
@@ -0,0 +1,747 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/ShippingAddress.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/Status.java [hunk 5]
```diff
@@ -0,0 +1,290 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-api/src/main/java/com/liferay/headless/commerce/admin/shipment/dto/v1_0/Status.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/Error.java [hunk 6]
```diff
@@ -0,0 +1,152 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/Error.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/Shipment.java [hunk 7]
```diff
@@ -0,0 +1,427 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/Shipment.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/ShipmentItem.java [hunk 8]
```diff
@@ -0,0 +1,258 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/ShipmentItem.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/ShippingAddress.java [hunk 9]
```diff
@@ -0,0 +1,354 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/ShippingAddress.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/Status.java [hunk 10]
```diff
@@ -0,0 +1,129 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-client/src/main/java/com/liferay/headless/commerce/admin/shipment/client/dto/v1_0/Status.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShipmentDTOConverter.java [hunk 11]
```diff
@@ -0,0 +1,73 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShipmentDTOConverter.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShipmentItemDTOConverter.java [hunk 12]
```diff
@@ -0,0 +1,69 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShipmentItemDTOConverter.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShippingAddressDTOConverter.java [hunk 13]
```diff
@@ -0,0 +1,90 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/dto/v1_0/converter/ShippingAddressDTOConverter.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/graphql/servlet/v1_0/ServletDataImpl.java [hunk 14]
```diff
@@ -0,0 +1,85 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/graphql/servlet/v1_0/ServletDataImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/ShipmentItemResourceImpl.java [hunk 15]
```diff
@@ -0,0 +1,173 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/ShipmentItemResourceImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/ShippingAddressResourceImpl.java [hunk 16]
```diff
@@ -0,0 +1,110 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/ShippingAddressResourceImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/factory/ShipmentItemResourceFactoryImpl.java [hunk 17]
```diff
@@ -0,0 +1,244 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/factory/ShipmentItemResourceFactoryImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/factory/ShipmentResourceFactoryImpl.java [hunk 18]
```diff
@@ -0,0 +1,242 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/factory/ShipmentResourceFactoryImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/factory/ShippingAddressResourceFactoryImpl.java [hunk 19]
```diff
@@ -0,0 +1,245 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/factory/ShippingAddressResourceFactoryImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_element_set.jsp [hunk 20]
```diff
@@ -82,34 +82,55 @@ renderResponse.setTitle((structure == null) ? LanguageUtil.get(request, "new-ele
 		<aui:input name="groupId" type="hidden" value="<%= groupId %>" />
 		<aui:input name="structureId" type="hidden" value="<%= structureId %>" />
 		<aui:input name="structureKey" type="hidden" value="<%= structureKey %>" />
-		<aui:input name="serializedFormBuilderContext" type="hidden" value="<%= serializedFormBuilderContext %>" />
+		<aui:input name="name" type="hidden" />
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_element_set.jsp:85]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_element_set.jsp [hunk 21]
```diff
@@ -82,34 +82,55 @@ renderResponse.setTitle((structure == null) ? LanguageUtil.get(request, "new-ele
...
+		<aui:input name="description" type="hidden" />
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_element_set.jsp:86]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_element_set.jsp [hunk 22]
```diff
@@ -82,34 +82,55 @@ renderResponse.setTitle((structure == null) ? LanguageUtil.get(request, "new-ele
...
+		<aui:input name="serializedFormBuilderContext" type="hidden" value="<%= formBuilderContextJSONObject %>" />
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_element_set.jsp:87]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 23]
```diff
@@ -130,9 +137,9 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 		<aui:input name="formInstanceId" type="hidden" value="<%= formInstanceId %>" />
 		<aui:input name="groupId" type="hidden" value="<%= groupId %>" />
 		<aui:input name="ddmStructureId" type="hidden" value="<%= ddmStructureId %>" />
-		<aui:input name="name" type="hidden" value="<%= ddmFormAdminDisplayContext.getFormLocalizedName(formInstance) %>" />
-		<aui:input name="description" type="hidden" value="<%= ddmFormAdminDisplayContext.getFormLocalizedDescription() %>" />
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:134]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 24]
```diff
@@ -130,9 +137,9 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
...
-		<aui:input name="serializedFormBuilderContext" type="hidden" value="<%= serializedFormBuilderContext %>" />
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:135]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 25]
```diff
@@ -130,9 +137,9 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
...
+		<aui:input name="name" type="hidden" value="<%= ddmFormAdminDisplayContext.getFormLocalizedNameJSONObject(formInstance) %>" />
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:140]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties [hunk 26]
```diff
@@ -273,6 +273,7 @@ number-placeholder=Number
 numeric-field-type-description=It accepts only numbers.
```
Comment (liferay-appsec) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties:273]: Check that added properties are either titles or sentences

File: modules/apps/oauth2-provider/oauth2-provider-api/src/main/resources/content/Language.properties [hunk 27]
```diff
@@ -4,6 +4,7 @@ CLIENT_CREDENTIALS=Client Credentials
 REFRESH_TOKEN=Refresh Token
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-api/src/main/resources/content/Language.properties:4]: Check that added properties are either titles or sentences

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp [hunk 28]
```diff
@@ -116,7 +116,7 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
 						<div class="allowedGrantType <%= cssClassesStr %>">
 							<c:choose>
 								<c:when test="<%= grantType.equals(GrantType.AUTHORIZATION_CODE) || grantType.equals(GrantType.AUTHORIZATION_CODE_PKCE) %>">
-									<aui:input checked="<%= checked %>" data="<%= data %>" label="<%= grantType.name() %>" name="<%= name %>" onchange='<%= liferayPortletResponse.getNamespace() + "updateTrustedApplicationSection();" %>' type="checkbox" />
+									<aui:input checked="<%= checked %>" data="<%= data %>" label="<%= grantType.name() %>" name="<%= name %>" onchange='<%= liferayPortletResponse.getNamespace() + "updateAdminOptionsApplicationSection();" %>' type="checkbox" />
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp:119]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp [hunk 29]
```diff
@@ -228,12 +228,25 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
 		<c:if test="<%= oAuth2AdminPortletDisplayContext.hasAddTrustedApplicationPermission() %>">
 			<clay:col
 				id='<%= liferayPortletResponse.getNamespace() + "trustedApplicationSection" %>'
-				lg="12"
+				lg="6"
 			>
 				<h3 class="sheet-subtitle"><liferay-ui:message key="trusted-application" /></h3>
 
 				<aui:field-wrapper>
-					<aui:input checked="<%= (oAuth2Application == null) ? false : oAuth2Application.isTrustedApplication() %>" helpMessage="trusted-application-help" id="trustedApplication" label="trusted-application" name="trustedApplication" type="checkbox" />
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp:236]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp [hunk 30]
```diff
@@ -228,12 +228,25 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
...
+		</c:if>
+
+		<c:if test="<%= oAuth2AdminPortletDisplayContext.hasRememberDevicePermission() %>">
+			<clay:col
+				id='<%= liferayPortletResponse.getNamespace() + "rememberDeviceSection" %>'
+				lg="6"
+			>
+				<h3 class="sheet-subtitle"><liferay-ui:message key="remember-device" /></h3>
+
+				<aui:field-wrapper>
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp:248]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/authorize/authorize.jsp [hunk 31]
```diff
@@ -105,6 +105,12 @@ if (Validator.isNotNull(replyTo) && !replyTo.startsWith(PortalUtil.getPortalURL(
 
 						</ul>
 
+						<c:if test="<%= oAuth2Application.isRememberDevice() %>">
+							<aui:field-wrapper>
+								<aui:input checked="<%= false %>" helpMessage="remember-device-help" id="rememberDevice" label="remember-device" name="rememberDevice" type="checkbox" />
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/authorize/authorize.jsp:110]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties [hunk 32]
```diff
@@ -85,6 +85,9 @@ redirect-uris=Callback URIs
 redirect-uris-help=Specify where to redirect the user after the authorization.
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties:85]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
ci:forward

**Comment by martamedio** [comment]
Wrong SF at BChan's origin, please see: https://github.com/liferay-appsec/liferay-portal/pull/306

**Pull request closed without merging**